メンテナンスモードONにした時にexcel_import_exportで不要なエラーが出る

### バグの説明
メンテナンスモードONにした時にexcel_import_exportで「UnBoundLocalError」が発生する。

### 該当ログ
```text
{"stream":"stdout","logtag":"F","log":"[2026-04-11 10:39:13,528][INFO] [ORGANIZATION_ID:XXX][WORKSPACE_ID:XXX][USER_ID:XXX] [timestamp=2026-04-11T01:39:13.521Z] exception_msg='cannot access local variable 'objdbca' where it is not associated with a value' (util.py:209)"}
{"stream":"stdout","logtag":"F","log":"[2026-04-11 10:39:13,534][ERROR] [ORGANIZATION_ID:XXX][WORKSPACE_ID:XXX][USER_ID:XXX] Traceback (most recent call last):"}
{"stream":"stdout","logtag":"F","log":"  File \"/exastro/common_libs/ci/util.py\", line 198, in organization_job"}
{"stream":"stdout","logtag":"F","log":"    main_logic_exec(organization_id, workspace_id)"}
{"stream":"stdout","logtag":"F","log":"  File \"/exastro/backyard_main.py\", line 399, in backyard_main"}
{"stream":"stdout","logtag":"F","log":"    objdbca.db_disconnect()"}
{"stream":"stdout","logtag":"F","log":"    ^^^^^^^"}
{"stream":"stdout","logtag":"F","log":"UnboundLocalError: cannot access local variable 'objdbca' where it is not associated with a value"}

--- a/ita_root/ita_by_ansible_execute/backyard_main.py
+++ b/ita_root/ita_by_ansible_execute/backyard_main.py
@@ -277,7 +277,9 @@ def execute_control(common_db, all_execution_limit, org_execution_limit):
             # organizationに実行中の処理があるか確認
             if rec["ORGANIZATION_ID"] not in org_exec_count_list:
                 org_exec_count_list[rec["ORGANIZATION_ID"]] = 0
-    return execution_list, all_exec_count, org_exec_count_list, records
+        return execution_list, all_exec_count, org_exec_count_list, records
+    else:
+        return [], 0, {}, []
 
 
 def child_process_exist_check(common_db, target_shema, ansibleAg):
@@ -518,7 +520,9 @@ def get_running_process(common_db, target_shema):
             if len(target_shema) > count:
                 sql += "UNION ALL "
         records = common_db.sql_execute(sql, [])
-    return records
+        return records
+    else:
+        return []
 
 
 def get_now_datetime(format='%Y/%m/%d %H:%M:%S', type='str'):

--- a/ita_root/ita_by_excel_export_import/backyard_main.py
+++ b/ita_root/ita_by_excel_export_import/backyard_main.py
@@ -52,6 +52,9 @@ def backyard_main(organization_id, workspace_id):
     debug_msg = g.appmsg.get_log_message("BKY-20001", [])
     g.applogger.debug(debug_msg)
 
+    # メンテナンス中にobjdbcaが未定義である旨が出力されるため、Noneで予め定義しておく
+    objdbca = None
+
     try:
         # インポート実行用のアップロードID
         upload_id = ""
@@ -396,6 +399,7 @@ def backyard_main(organization_id, workspace_id):
         # ステータスを完了(異常)に更新
         util.setStatus(task['EXECUTION_NO'], STATUS_FAILURE, objdbca)
     finally:
-        objdbca.db_disconnect()
+        if objdbca is not None:
+            objdbca.db_disconnect()
 
     return

